src/test/test_bitcoin.cpp

```diff
@@ -25,7 +25,7 @@
 
 CClientUIInterface uiInterface; // Declared but not defined in ui_interface.h
 CWallet* pwalletMain;
-libzerocash::ZerocashParams *pzerocashParams;
+ZCJoinSplit *pzcashParams;
```

Comment:
Renaming these variables to include "test" would set my mind at ease about whether they could be confused with the ones in `init.cpp` (either by the compiler/linker or by a human reader of the code).